src/vs/code/electron-main/app.ts

```diff
@@ -225,12 +225,15 @@ export class CodeApplication extends Disposable {
 				this.nativeHostMainService?.openExternal(undefined, url);
 			});
 
-			session.defaultSession.setPermissionRequestHandler((webContents, permission /* 'media' | 'geolocation' | 'notifications' | 'midiSysex' | 'pointerLock' | 'fullscreen' | 'openExternal' */, callback) => {
+			session.defaultSession.setPermissionRequestHandler((_webContents, permission /* 'media' | 'geolocation' | 'notifications' | 'midiSysex' | 'pointerLock' | 'fullscreen' | 'openExternal' */, callback, details) => {
+				if (permission === 'clipboard-read') {
+					return callback(true);
+				}
 				return callback(false);
```

Comment:
```suggestion
				return callback(permission === 'clipboard-read');
```

Reply:
This part of the change is needed for iframe based webviews. Our normal webviews have their own session that we handle separately. I can add a basic check here without too much trouble. The more complete check I mentioned would require knowing which webviews are actually registered